test/integration/system_install/controls/system_install.rb

```diff
@@ -1,18 +1,16 @@
 # frozen_string_literal: true
-global_ruby = '2.3.4'
 
-control 'Rbenv should be installed' do
-  title 'Rbenv should be installed globally'
+control 'Rbenv system insall' do
+  title 'Rbenv should be installed system wide'
 
-  desc "Can set global Ruby version to #{global_ruby}"
+  desc 'Rbenv should be installed and run successfully'
   describe bash('source /etc/profile.d/rbenv.sh && rbenv versions --bare') do
     its('exit_status') { should eq 0 }
-    its('stdout') { should include(global_ruby) }
```

Comment:
This is what I added back then, because I thought the recipe installs a Ruby version, but it doesn't. I also changed the `desc` accordingly.